packages/video_player/video_player_android/android/build.gradle

```diff
@@ -57,6 +57,7 @@ android {
         unitTests.includeAndroidResources = true
         unitTests.returnDefaultValues = true
         unitTests.all {
+            jvmArgs "-Xmx1g"
```

Comment:
Add a comment here similar to https://github.com/flutter/packages/pull/7805/files

```
            // The org.gradle.jvmargs property that may be set in gradle.properties does not impact
            // the Java heap size when running the Android unit tests. The following property here
            // sets the heap size to a size large enough to run the robolectric tests across
            // multiple SDK levels.
```